### PR TITLE
Update App Icon

### PR DESCRIPTION
**Pull request by joshmcarthur**
This pull request updates the icons for the apps to match the logos provided by Rabid for NZSL to use across their website and social media platforms.

It upgrades the existing icon sizes and provides a higher-resolution default icon (since it's better to have a higher-resolution fallback than lower). 

I used the built-in image size generator from Android Studio, which generated files into the `mipmap`, rather than `drawable` folders. I was confused by this, but had a read of https://android-developers.googleblog.com/2014/10/getting-your-apps-ready-for-nexus-6-and.html, which clarified the rules:

> It’s best practice to place your app icons in mipmap- folders (not the drawable- folders) because they are used at resolutions different from the device’s current density. For example, an xxxhdpi app icon can be used on the launcher for an xxhdpi device.

In other words, it sounds like Google's hardware designers picked a density that fell between two classes of drawable, and so they now recommend placing launcher icons in the mipmap folder as different launchers may use different images regardless of the screen density.

### CODE REVIEW

**Comment by Br3nda** [comment]
travis is very unhappy about this.

**Comment by joshmcarthur** [comment]
@Br3nda I enabled Travis some time ago and do have a .travis.yml in a feature branch, however I never did get the build working. Should we disable the integration until it is working?
